bpf/action.h

```diff
@@ -566,8 +576,11 @@ static int tail_action_set_masked(struct __sk_buff *skb)
          */
         set_ip_tos(skb, ipv4->ipv4_tos);
         set_ip_ttl(skb, ipv4->ipv4_ttl);
+
+#if ENABLE_POINTER_LOOKUP
         set_ip_src(skb, ipv4->ipv4_src);
         set_ip_dst(skb, ipv4->ipv4_dst);
+#endif
```

Comment:
I can't understand why needs this two lines here, could you explain a little bit?

Reply:
these set the ip_src and ip_dst, then update the checksum.
if not enable_pointer_lookup and call these two, we hit some stack limitation issue.